src/inference/rejection.js

```diff
@@ -22,6 +21,17 @@ module.exports = function(env) {
       maxScore: 0,
       incremental: false
     });
+
+    this.throwOnError = options.throwOnError !== undefined ? options.throwOnError : true;
+    // the value of options.probe is the min sample rate
+    this.probe = options.probe;
```

Comment:
As you make progress on making the things that "default infer" is implemented on top of more independent, I think it will make sense to replace these generic `probe` options with something more specific. e.g. Here, rejection could have a `minSampleRate` option.